Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityInputActionsProfile.asset

```diff
@@ -21,7 +21,7 @@ MonoBehaviour:
   - id: 3
     description: Grip
     axisConstraint: 7
-  pointerAction:
-    id: 0
+  defaultPointerAction:
```

Comment:
Still not sure we need this separate action.

Reply:
I def think we should. I was actually contemplating adding a default up/down/and click action for all pointers.

Reply:
The reason _why_ we specifically need them is because of the way the Unity Canvas system works and how our need to raycast against the canvases and physical objects.

Take a look at the `MixedRealityInputManager`'s Raise Pointer methods.  In each we have to make sure we've got a valid `GraphicInputEventData` before we can properly execute the pointer up/down/click.

```
public void RaisePointerUp(IMixedRealityPointer pointer, Handedness handedness, IMixedRealityInputAction inputAction)
{
    // Create input event
    pointerEventData.Initialize(pointer.InputSourceParent, handedness, inputAction);

    if (MixedRealityManager.Instance.ActiveProfile.InputActionsProfile.DefaultPointerAction.Id == inputAction.Id)
    {
        ExecutePointerUp(HandlePointerUp(pointer));
    }
}
```

Reply:
Then this should be a static in the InputSystem that derives from the Currently configured pointer action.  Unless you feel this is a different action to the one I would configure on the spatialPointer

Reply:
That's a possibility.  Let me chew on this a bit more and see if I can come up with a better solution.

Reply:
Ended up removing for now until we need it again. After testing this out, I concur, we don't really need it